### ISSUE TITLE
New PR installs as QGIS.app not QGIS3.16.app

### ISSUE THREAD

**Issue by jlayt** [qgis/QGIS-Mac-Packager#100]
Previous releases installed the QGIS app with a version number in the name, i.e. QGIS3.14.app. The new package does not. This was very useful to allow parallel installs for testing and development purposes. It would be useful to have this back by default, with 50 desktops to support it makes my life easier!

**Comment by PeterPetrik** [comment]
Hi, I do not know other application that uses version prefix in the installation folder so we moved to more natural naming for MacOS. On the positive side, you can now move the application/rename the folder to whatever name you like.

**Comment by jlayt** [comment]
Fair enough, but would you at least consider calling it QGIS3.app? It was a life-saver for us when running both QGIS 2 and 3 in parallel while we switched over, and it would make the QGIS 4 migration easier when it comes.  We use Munki to manage our installs so I've set it to rename in there anyway, but it is very useful for testing purposes :-)
<img width="475" alt="Screenshot 2020-11-26 at 12 43 24" src="https://user-images.githubusercontent.com/3813093/100353022-c7bcfd80-2fe5-11eb-84ad-a7a0f39c4e2a.png">

**Comment by PeterPetrik** [comment]
qgis3 could work

**Comment by 3nids** [comment]
I would be in favor of reverting. It is pretty useful to have the possibility of having several versions at the same time.

**Comment by mapninja** [comment]
Ditto, I find it useful to run several versions, and was having some trouble with renaming manually, after the change.

**Comment by dkav** [comment]
I like to advocate maintaining the new naming convention (QGIS.app). I agree with @PeterPetrik original logic - I have only two apps that use a version number in their name.

**Comment by PeterPetrik** [comment]
I propose:

LTR will be 

`/Applications/QGIS.app`

and PR
`/Applications/QGIS3.18.app`

and nightly
`/Applications/QGIS3.19.app`

ok?

**Comment by dkav** [comment]
I use Homebrew to cask install QGIS, so my preference would be to have the PR as the main release. Separate casks for LTR and Nightly could be made (dependent on them being accepted). From what I can tell this follows common practice with Homebrew. For example, four versions of **Firefox** can be installed:

- [PR Release](https://formulae.brew.sh/cask/firefox#default) (`Firefox.app`)
- [Extended Support Release (ESR)](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/firefox-esr.rb)(`Firefox.app`)
- [Beta](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/firefox-beta.rb) (`Firefox.app`)
- [Developer Edition](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/firefox-developer-edition.rb) (`Firefox Developer Edition.app`)
- [Nightly](https://github.com/Homebrew/homebrew-cask-versions/blob/master/Casks/firefox-nightly.rb) (`Firefox Nightly.app`) 

Following this paradigm for **QGIS**, my vote would be for:

- PR `QGIS.app`
- LTR `QGIS.app`
- Nightly `QGIS Nightly.app`
